cmd/describe/machinepool/cmd.go

```diff
@@ -71,6 +71,7 @@ func DescribeMachinePoolRunner(userOptions DescribeMachinepoolUserOptions) rosa.
 			userOptions.machinepool = argv[0]
```

Comment:
@hunterkepley I think you need to replace the `cobra.NoArgs` on the `cmd` definition with `cobra.MaximumNArgs(1)`. That's what is was prior to your refactor and why passing the `workers` name as an arg is being rejected.

Reply:
@robpblake fixed